### DISCUSSION TITLE
Failed to conenect SCTP socket

### DISCUSSION THREAD

**Discussion by carlosgcg00** [srsran/srsRAN_Project#159] [General Help]
Hi everyone,

I'm having trouble with the installation of srsran. I've followed the instructions in the guide: https://docs.srsran.com/projects/project/en/latest/user_manuals/source/installation.html, but when I try to execute the gnb, I get the following error message:

**Connecting to AMF on 127.0.0.1:38412
Failed to connect SCTP socket to 127.0.0.1:38412. error="Connection refused" timeout=0ms**

I'm not sure what to do. Can anyone help?

Thanks in advance

**Comment by pgawlowicz** [comment]
You need to start 5G Core network, for example Open5GS.
Please check this [tutorial](https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html).
